feat : create Reusable CustomModal, CustomAlert

frankkim92 opened the pull request:
## What is this PR? :mag:
재사용 가능한 CustomModal을 사용할 수 있습니다.
재사용 가능한 CustomAlert(sweetAlert2)을 사용할 수 있습니다.

## branch

- feat/modal-component -> develop

## Changes :memo:

- 재사용가능한 모달을 사용할 수 있습니다.
- 재사용가능한 커스텀알럿(sweetAlert2 라이브러리)을 사용할 수 있습니다.
- util폴더에있는 index.ts에서 커스텀 알럿함수를 export 하여 사용할 수 있습니다.
- 커스텀 버튼에 cursor pointer를 추가했습니다.
- 커스텀 인풋에서 box-shadow를 제거했습니다.
- 커스텀 버튼에서 box-shadow를 제거했습니다.



by @frankkim92 
@Jeremy-Kr 
@harenohee 
@ambition0103 
@dollorofkorea

frankkim92 committed:
feat : Update Reusable CustomModal을

재사용 가능한 CustomModal을 생성하였습니다.
@by frankkim92

Jeremy-Kr commented on src/components/common/CustomButton.tsx:
```diff
@@ -15,7 +15,7 @@ interface ButtonProps {
 export const CustomButton = styled.button<ButtonProps>`
   border: 0.5px solid gray;
   box-shadow: 0px 2px 2px lightgray;
```
요고는 남기기로 했나요? 제가 볼때는 일괄적으로 그냥 빼는것도 좋아보입니다.

frankkim92 replied:
@Jeremy-Kr 넵 바로 빼도록 하겠습니다.

frankkim92 replied:
제거 완료하여, 커밋 올렸습니다 !

Jeremy-Kr commented on src/utils/index.ts:
```diff
@@ -1 +1,32 @@
+import Swal from 'sweetalert2';
+
+// 삭제버튼 커스텀 알럿
+export const handleDeleteButtonClick = () => {
+  Swal.fire({
+    title: '정말 삭제하시겠습니까?',
+    text: '다시 한 번 생각해보세요.',
+    icon: 'warning',
+    showCancelButton: true,
+    confirmButtonColor: '#3085d6',
+    cancelButtonColor: '#d33',
+    confirmButtonText: 'Ok',
+    // id값이 맞아야 삭제가 되는데.....
+  }).then((result) => {
+    if (result.isConfirmed) {
+      Swal.fire(
+        '삭제가 완료되었습니다.',
+        'Your file has been deleted.',
+        'success'
+      );
+    }
+  });
+};
+
+// 경고 커스텀 알럿 입니다.
+export const handleAlertButtonClick = () => {
+  Swal.fire({
+    icon: 'warning',
+    title: '경고입니다. 돌아가세요',
+    confirmButtonColor: '#3085d6',
+  });
+};
```
재사용 가능한 함수로 변경하는게 어려우실까요? 어려우시면 제가 작업진행하겠습니다

Jeremy-Kr reviewed (changes requested):
리뷰 확인 부탁드려요

frankkim92 committed:
fix : Update Reusable CustomModal

- 재사용 가능한 CustomModal을 생성하였습니다.
- CustomInput 에서 box-shadow 효과를 제거했습니다.
@by frankkim92

Jeremy-Kr committed:
Merge branch 'dev' into feat/modal-component
